README.md

```diff
@@ -19,65 +42,34 @@ Mautic Introduction
 
-* = Security Support for 2.16 will only be provided for Mautic itself, not for core dependencies that are EOL like Symfony 2.8.
-
-## Getting Started
-
-The GitHub version is recommended for development or testing. Production package ready for install with all the libraries is at [https://www.mautic.org/download](https://www.mautic.org/download).
-
-Documentation on how to use Mautic is available at [https://docs.mautic.org](https://docs.mautic.org).
-
-This is a simple 3 step installation process. You'll want to make sure you already have [Composer v1](http://getcomposer.org) available on your computer as this is a development release and you'll need to use Composer to download the vendor packages. Note that v2 is not yet supported.
-
-<table width="100%" border="0">
-	<tr>
-		<td>
-			<center><b>Step 1</b></center>
-		</td>
-		<td>
-			<center><b>Step 2</b></center>
-		</td>
-		<td>
-			<center><b>Step 3</b></center>
-		</td>
-	</tr>
-	<tr>
-		<td align="center" width="33.3%">
-			<a href="https://github.com/mautic/mautic/archive/master.zip">Download the repository zip</a><br />Extract this zip to your web root.
-		</td>
-		<td align="center" width="33.3%">
-			Run the following command to install required packages.<br /> <code>composer install</code>
-		</td>
-		<td align="center" width="33.3%">
-			Open your browser and complete the installation through the web installer.
-		</td>
-	</tr>
-</table>
+`*`Security support for 2.16 will only be provided for Mautic itself, not for core dependencies that are EOL, such as Symfony 2.8.
 
-**Get stuck?** *No problem. Check out [general troubleshooting](https://docs.mautic.org/en/troubleshooting) and if it won't solve your issue join us at the <a href="https://www.mautic.org/community">Mautic community</a> for help and answers.*
+## Software Downloads
+The GitHub version is recommended for both development and testing. The production package (including all libraries) is available at [mautic.org/download](https://www.mautic.org/download).
 
-## Disclaimer
-Installing from source is only recommended if you are comfortable using the command line. You'll be required to use various CLI commands to get Mautic working and to keep it working. If the source and/or database schema gets out of sync with Mautic's releases, the release updater may not work and will require manual updates. For production the pre-packaged Mautic available at [mautic.org/download](https://www.mautic.org/download) is recommended.
+## Installation
+### Disclaimer
+*Install from source only if you are comfortable using the command line. You'll be required to use various CLI commands to get Mautic working and keep it working. If the source/database schema gets out of sync with Mautic releases, the release updater may not work and will require manual updates. For production, we recommend the pre-packaged Mautic which is available at [mautic.org/download](https://www.mautic.org/download).*
```

Comment:
Refactor to use existing link macro that was used in my edit above (reduces duplication!)

```suggestion
*Install from source only if you are comfortable using the command line. You'll be required to use various CLI commands to get Mautic working and keep it working. If the source/database schema gets out of sync with Mautic releases, the release updater may not work and will require manual updates. For production, we recommend the pre-packaged Mautic which is available at [mautic.org/download][download-mautic].*
```